tests/debuginfo/basic-types-globals-metadata.rs

```diff
@@ -1,5 +1,11 @@
 //@ compile-flags:-g
 
+// Targets with broken f16 cannot link due to missing __gnu_f2h_ieee
+// See https://github.com/llvm/llvm-project/issues/97981
+// and https://github.com/rust-lang/compiler-builtins/issues/655
+//@ ignore-powerpc
+//@ ignore-powerpc64
```

Comment:
I feel that we really should not have types that must be ignored in the same file as other types.

Reply:
I included a workaround instead of disabling the tests.

Reply:
...huh.

Reply:
my guess is that it tries to convert the f16 to the f32 due to rules about how we represent the tuple layout